### DISCUSSION TITLE
Ads format / DoQ support

### DISCUSSION THREAD

**Discussion by LexterS999** [0xERR0R/blocky#234] [Ideas]
Hey, what's ads block format supported by Blocky, it's important to me? I think good idea to add regex filters too
Also take a look at DoQ format as alternative of DoH, it's better/safer/faster. Now only aguard dns / adguard home support it

P/S when new version of Blocky will be released, and I guess will be added new instruction for caching section, right?

**Comment by 0xERR0R** [comment]
Blocky supports only a simple list of domains (one domain/IP per row) or hosts format

**Reply by 0xERR0R** [reply]
Regarding the DoQ protocol -> is already tracked here  #148

**Comment by LexterS999** [comment]
Yeah, DoQ is amazing for speed and safety.

P/S when new version of Blocky will be released, and I guess will be added new instruction for caching section, right?

**Comment by LexterS999** [comment]
Also, I think it will be better to make a universal format for logs. It's hard, always go to the ftp to see what's blocking and not blocking. And now logs format hard to use via ssh terminal. I think it must be with the name of user like: logs-testuser.txt

What do you think?

**Comment by 0xERR0R** [comment]
Regex support is now released (#12).

**Comment by LexterS999** [comment]
When QUIC support will be released?

https://github.com/lucas-clemente/quic-go
https://github.com/devsisters/goquic
https://github.com/cloudflare/quiche

**Reply by 0xERR0R** [reply]
There is an open issue (#148) to track it. It will be definitely implemented, but I can't give you some ETAs.
